tensorflow/compiler/tests/image_ops_jit_compile_test.py

```diff
@@ -14,21 +14,27 @@
 # ==============================================================================
```

Comment:
This test doesn't run at least internally due to missing BUILD dependencies here - 

https://github.com/tensorflow/tensorflow/blob/74cb43c6e9821cfbff259f243d6d31f9c10308bb/tensorflow/compiler/tests/BUILD#L2551

Mind adding those here please?